### PR TITLE
Home 11

### PR DESCRIPTION
**Pull request by barabule4ka**
Test

### CODE REVIEW

File: home_11_SeleniumTests/Tests/CalcTests.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using MTS_SeleniumTests.Calculate;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+//namespace home_11_SeleniumTests.Tests
```
Comment (VladimirVolkovVD) [inline home_11_SeleniumTests/Tests/CalcTests.cs:8]: uncomment or delete file at all

File: home_11_SeleniumTests/Tests/BaseTest.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        public void OneTimeSetUp()
+        {
+            calculator = new Calculator();
+        }
+
+        //[SetUp]
+        //public void Setup()
+        //{
+        //    Console.WriteLine("---- SetUp");
+        //}
```
Comment (VladimirVolkovVD) [inline home_11_SeleniumTests/Tests/BaseTest.cs:28]: uncomment or delete

Comment (barabule4ka) [reply home_11_SeleniumTests/Tests/BaseTest.cs:28]: Deleted

File: home_11_SeleniumTests/Tests/CalcTests.cs (hunk 1 above)
Comment (barabule4ka) [reply home_11_SeleniumTests/Tests/CalcTests.cs:8]: Deleted

**Review by VladimirVolkovD — APPROVED** [review]

**Pull request merged**